src/main/scala/ch/epfl/bluebrain/nexus/kg/resources/attachment/AttachmentStore.scala

```diff
@@ -177,7 +177,7 @@ object AttachmentStore {
           val relIri = Iri.relative(s"${id.parent.id}/${uuid.takeWhile(_ != '-').mkString("/")}/$uuid")
           EitherT.fromEither[F](relIri.left.map[Rejection](Unexpected).flatMap { relative =>
             Try {
-              val attachmentPath = new File(new File(base.show), relative.show).toPath
+              val attachmentPath = new File(new File(new URI(base.show)), relative.show).toPath
```

Comment:
maybe we should do `new File(base.path.show, relative.show).toPath` instead; I don't think query params or fragments have a place in the binary location.

Reply:
I agree, I think using `IRI`s here unnecessarily complicates things. We should just stick with `Path`s.